lib/SerializerStream.js

```diff
@@ -70,4 +72,10 @@ class SerializerStream extends Readable {
   }
 }
 
+function isXsdInteger (object) {
+  return object.datatype &&
+    object.datatype.value === 'http://www.w3.org/2001/XMLSchema#integer' &&
+    !isNaN(object.value)
```

Comment:
This check is insufficient because `object.value` could contain a decimal value or be longer than 20 digits in length:

https://www.w3.org/TR/json-ld11-api/#data-round-tripping

I think this might suffice:

```suggestion
    !isNaN(object.value) && parseInt(object.value, 10).toFixed(0) === object.value
```

Reply:
Ah good catch, but let me add a test for this as well

Reply:
anything left to do?

Reply:
LGTM.